planetiler-custommap/src/main/java/com/onthegomap/planetiler/custommap/ConfiguredMapMain.java

```diff
@@ -0,0 +1,76 @@
+package com.onthegomap.planetiler.custommap;
+
+import com.onthegomap.planetiler.Planetiler;
+import com.onthegomap.planetiler.config.Arguments;
+import com.onthegomap.planetiler.custommap.configschema.DataSource;
+import com.onthegomap.planetiler.custommap.configschema.DataSourceType;
+import com.onthegomap.planetiler.custommap.configschema.SchemaConfig;
+import java.io.File;
+import java.io.FileInputStream;
+import java.net.URI;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collection;
+import org.yaml.snakeyaml.Yaml;
+
+public class ConfiguredMapMain {
+
+  /*
+   * Main entrypoint
+   */
+  public static void main(String[] args) throws Exception {
+    run(Arguments.fromArgsOrConfigFile(args));
+  }
+
+  private static void run(Arguments args) throws Exception {
+    Path dataDir = Path.of("data");
+    Path sourcesDir = dataDir.resolve("sources");
+
+    String schemaFile = args.getString(
+      "schema",
+      "Location of YML-format schema definition file",
+      "");
```

Comment:
Why not `args.inputFile(...)` ? That does a check to ensure it exists as well.  You could also add an `inputFile` variation without a default value that will fail-fast if the argument is not specified.

Reply:
Resolved in fa7fe2f

Reply:
Resolved in fa7fe2f